spec/javascripts/ga_cross_domain_form_tracking_spec.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env jasmine, jquery */
+/* global attachCrossDomainTrackerToInput */
+
+describe('attachCrossDomainTrackerToInput', function () {
+  var form
+  var tracker
+  var linker
+
+  beforeEach(function () {
+    form = $('<form method="POST" action="/somewhere" class="account-signup">' +
+                '<input type="hidden" name="key" value="value" />' +
+                '<button type="submit">Create a GOV.UK account</button>' +
+              '</form>')
+
+    window.ga = {
+      getAll: function () {}
+    }
+
+    window.gaplugins = {
+      Linker: function () {}
+    }
+
+    linker = {
+      decorate: function () {}
+    }
+
+    spyOn(window.ga, 'getAll').and.returnValue([])
+    spyOn(window.gaplugins, 'Linker').and.returnValue(linker)
+    spyOn(linker, 'decorate').and.returnValue('/somewhere?_ga=abc123')
+  })
+
+  afterEach(function () {
+    form.remove()
+  })
+
+  it('leaves the form action unchanged if ga is not present', function () {
+    tracker = []
+    attachCrossDomainTrackerToInput(form, tracker)
+    expect(form.attr('action')).toEqual('/somewhere')
+  })
+
+  it('leaves the form action unchanged there are no trackers in ga', function () {
```

Comment:
Am I missing something or is this identical to the previous test?

Reply:
ah, humm yeah initially we were initalising GA within the test... and we were going to have a test to show that the code doesn't fire if GA isn't around.

But this might be an issue here...